<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainClient</class>
 <widget class="QDialog" name="MainClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>绑定</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>同步目录：</string>
   </property>
  </widget>
  <widget class="QLabel" name="syncPathLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;未选择&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>解除绑定</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="logBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>831</width>
     <height>421</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="source">
    <url>
     <string>file:///E:/test.txt</string>
    </url>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="downloadInfoLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前下载：</string>
   </property>
  </widget>
  <widget class="QLabel" name="uploadInfoLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前上传：</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="downloadBar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="uploadBar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
